includes/documents/class-wcpdf-invoice.php

```diff
@@ -146,8 +153,11 @@ public function get_sequential_number_store() {
 	public function get_sequential_number_store_name( $date ) {
 		$year       = $date->date_i18n( 'Y' );
 		$store_name = "{$this->slug}_number_{$year}";
-	
-		return apply_filters( "wpo_wcpdf_{$this->slug}_store_name", $store_name, $this );
+
+		// legacy filter for backward compatibility
+		$store_name = apply_filters( 'wpo_wcpdf_document_sequential_number_store', $store_name, $this );
```

Comment:
This should be filtered before apending the `_{$year}`, otherwise it breaks current implementations.